.circleci/config.yml

```diff
@@ -33,7 +33,7 @@ jobs:
           command: |
             /opt/circleci/.nvm/nvm.sh use 8.5.0
             cd ui
-            ng test --watch=false -sm=false
+            ng test --watch=false -sm=false --browsers=ChromeHeadless
```

Comment:
tiny nit: Would you mind actually change `--watch=false` to `--single-run`. I added the watch flag but I think single run is much clearer.

Reply:
Done.